.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,53 @@
+# External variables Settable at queue time:
```

Comment:
Can you please move this into the eng/pipelines directory? https://github.com/Azure/azure-sdk-for-java/tree/master/eng/pipelines That is where we are starting to put all yml files for the other language repos.

Reply:
btw.. why is the mgmt yaml supposed to be in eng/pipeline if the rest are still under https://github.com/Azure/azure-sdk-for-java/tree/master/.azure-pipelines?

Are all of these eventually being moved out there?

Reply:
@mitchdenny is working on moving all the others out of the `.azure-pipelines` directory and into `eng/pipeline`